Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN apt-get --no-install-recommends -y install \
     libmozjs185-dev \
     python
 
+# Install Nodejs
+RUN curl -sL https://deb.nodesource.com/setup | bash -	
```

Comment:
or we could just borrow a few bits from https://github.com/klaemo/docker-couchdb/blob/master/2.0-dev/Dockerfile#L32-L35 if @klaemo don't mind (:

P.S. @janl, another question is a reason to maintain our Dockerfile when @klaemo keep his well. May be we could unite our effort on this with Clemens instead of doing the same work twice?